.gitignore

```diff
@@ -1,18 +1,4 @@
 # Don’t commit the following directories created by pub.
-.pub
-build/
+.dart_tool/
 .packages
-packages
-
-# Or the files created by dart2js.
-*.dart.js
-*.dart.precompiled.js
-*.js_
-*.js.deps
-*.js.map
-*.sw?
-.idea/
```

Comment:
You want it to commit the .idea directory?

Reply:
No. I want users to put it in their `.gitignore_global` file.

Reply:
See https://www.dartlang.org/guides/libraries/private-files

Reply:
Ah, thanks.